### PR TITLE
Show failed services 'docker-compose start' when containers are not availabe

### LINKED ISSUE #5469: Docker-compose start strange behviour when containers are not available
When I use `docker-compose start ` before a `docker-compose up` I obtain the following result:
```
Starting mongo         ... done
Starting elasticsearch ... done
Starting transporter   ... done
Starting pdf           ... done
Starting back          ... done
Starting front         ... done
ERROR: No containers to start
```
This behaviour is pretty strange as the output is contradictory it says that it start all containers but the errors say the opposite.

### PR DESCRIPTION
**Pull request by collin5**
Resolves #5469

### CODE REVIEW

File: compose/parallel.py [hunk 1]
```diff
@@ -49,8 +49,12 @@ def parallel_execute_watch(events, writer, errors, results, msg, get_name):
     """
     error_to_reraise = None
     for obj, result, exception in events:
+        from compose.service import Service # noqa
         if exception is None:
-            writer.write(msg, get_name(obj), 'done', green)
+            if isinstance(obj, Service) and not obj.containers():
```
Comment (shin-) [inline compose/parallel.py]: Can we use duck-typing here instead and avoid the in-block import? e.g.

```python
if hasattr(obj, 'containers') and callable(obj.containers) and not obj.containers():
  ...
```

**Comment by collin5** [comment]
Fixed. Thanks!

File: compose/parallel.py [hunk 2]
```diff
@@ -50,7 +50,10 @@ def parallel_execute_watch(events, writer, errors, results, msg, get_name):
     error_to_reraise = None
     for obj, result, exception in events:
         if exception is None:
-            writer.write(msg, get_name(obj), 'done', green)
+            if callable(getattr(obj, 'containers', None)) and not obj.containers():
```
Comment (shin-) [inline compose/parallel.py]: Sorry, one last thing - can you add a comment here to explain what problem this solves, as a reminder for our future selves? :)

**Comment by collin5** [comment]
Fixed! Thanks 👍

**Pull request merged**